### ISSUE TITLE
Split out the country from the county

### ISSUE THREAD

**Issue by DeniseColbert** [FreeUKGen/FreeCENMigration#598]
It is a real pain to scroll down to Scotland, I'd like to filter to just Scotland counties with just one click and not all this scrolling.

**Comment by PatReynolds** [comment]
Closing as duplicates #467

**Comment by PatReynolds** [comment]
No, may be different - may want list of Scottish counties to select from

**Comment by richpomfret** [comment]
To ensure we are consistent with this (not just Scottish counties).
